StRoot/StRefMultCorr/BadRunSubSys.h

```diff
@@ -0,0 +1,38 @@
4,20070047,20070041,20067029,20067028,20092005,20067024,20067023,20080020,20063011,20075025,20075020,20071037,20071036,20068034,20088012,20065056,20074043,20074045,20074044,20074046,20081001,20079044,20060062,20060060,20060061,20082038,20082034,20076007,20076004,20076001,20078046,20078040,20078043,20091009,20091006,20091003,20057050,20089015,20089012,20089018,20068001,20084016,20068004,20083077,20084013,20068008,20085017,20064012,20064011,20078032,20090031,20078035,20074005,20074004,20074007,20074006,20074001,20077002,20074003,20074002,20093035,20074009,20074008,20092030,20092033,20092038,20079013,20079017,20065018,20065017,20082002,20082005,20073013,20073017,20069020,20069023,20069026,20066019,20066015,20087007,20072012,20072016,20083032,20087008,20067012,20067016,20067014,20067015,20067019,20084022,20066066,20071001,20066068,20075035,20075031,20071006,20071007,20071004,20071005,20071003,20075039,20066067,20074034,20074030,20074032,20074033,20082047,20079020,20079023,20076037,20076034,20081025,20083079,20091019,20091016,20072050,20091012,20083074,20089009,20089008,20057025,20057026,20089006,20068012,20085009,20069010,20068019,20085006,20078028,20078022,20071041,20090021,20071047,20071044,20066026,20066023,20062036,20069060,20093001,20093005,20067041,20067045,20067047,20067046,20092027,20077005,20092024,20063034,20063035,20092021,20063039,20077008,20075048,20075043,20088033,20088030,20088037,20070043,20070042,20086005,20086002,20082013,20082010,20082016,20082019,20073025,20073022,20069054,20069057,20069050,20069053,20078067,20078063,20078060,20072002,20083022,20083025,20083024,20072005,20083029,20072009,20063036,20067004,20080012,20067001,20067009,20080006,20075008,20075009,20075002,20075001,20075006,20075007,20075004,20080009,20090014,20071010,20071013,20090017,20090011,20071016,20071019,20068058,20068055,20068051,20074029,20074027,20074026,20074023,20074021,20074020,20070016,20070013,20070010,20092057,20092054,20092053,20070019,20075011,20075010,20076051,20076054,20076059,20082059,20082056,20082053,20082050,20076025,20076021,20081018,20060012,20081014,20081015,20081012,20076028,20091020,20072047,20072046,20072045,20072041,20086056,20071050,20076031,20068026,20069001,20069007,20069004,20071059,20078019,20078016,20071056,20078013,20071053,20083019,20062009,20062007,20092018,20093016,20067038,20093010,20092012,20067030,20092015,20077017,20077014,20077011,20064040,20077018,20058003,20071029,20058001,20090048,20075054,20075057,20058004,20071024,20071027,20070002,20088009,20088005,20072055,20086015,20086012,20086011,20082024,20082029,20091011,20076017,20076010,20076013,20069042,20078051,20078057,20078054,20087021,20072036,20072037,20072034,20072035,20072038,20072039,20089028,20089003,20084002,20084001,20084005,20084009,20080016,20075015,20075014,20064008,20064009,20075013,20090005,20090002,20071063,20090008,20068060,20068064,20074012,20074010,20074011,20074016,20074017,20074014,20058002,20093025,20074018,20074019,20070005,20058005,20075060,20075066};
```

Comment:
Judging from the way this array is used in the cxx file, I think there should not be any duplicated entries in the array. If this is true, please add a comment such that future users do not prepare the wrong format. At the moment, the injection runs and TPC bad runs do not overlap by design, as the TPC bad runs are determined after removing the injection runs. However, the bad run lists for different subsystems could in principle overlap, and one would need a dedicated macro or script to prepare the array from the lists of bad runs.

Reply:
I have a python script for preparing the correct format, I will add it to the next pull request